pdaf-code/make.arch/linux_gfortran.h

```diff
@@ -0,0 +1,62 @@
+######################################################
+# Include file with machine-specific definitions     #
+# for building PDAF.                                 #
+#                                                    #
+# Variant for Linux with gfortran and OpenMPI        #
+#                                                    #
+# In the case of compilation without MPI, a dummy    #
+# implementation of MPI, like provided in the        #
+# directory nullmpi/ has to be linked when building  #
+# an executable.                                     #
+######################################################
+# $Id: linux_gfortran_openmpi.h 1565 2015-02-28 17:04:41Z lnerger $
+
+
+# Compiler, Linker, and Archiver
+FC = mpif90
+LD = $(FC)
+AR = ar
+RANLIB = ranlib
+
+# C preprocessor
+# (only required, if preprocessing is not performed via the compiler)
+CPP = /usr/bin/cpp
+
+# Definitions for CPP
+# Define USE_PDAF to include PDAF
+# Define BLOCKING_MPI_EXCHANGE to use blocking MPI commands to exchange data between model and PDAF
+# Define PDAF_NO_UPDATE to deactivate the analysis step of the filter
+# (if the compiler does not support get_command_argument()
+# from Fortran 2003 you should define F77 here.)
+CPP_DEFS = #-DUSE_PDAF
```

Comment:
I believe this change wasn't meant to be included at this stage?
```suggestion
CPP_DEFS = -DUSE_PDAF
```